python/src/server.py

```diff
@@ -0,0 +1,53 @@
+# from typing import Any, List, Literal, Optional, Union
+
+# from pydantic import validator
+
+from konan_sdk.konan_service.services import KonanService
+from konan_sdk.konan_service.models import KonanServiceBaseModel
+from konan_sdk.konan_service.serializers import (
+    KonanServiceBasePredictionRequest, KonanServiceBasePredictionResponse)
+
+
+# TODO: [1] Define your request schema
+class MyPredictionRequest(KonanServiceBasePredictionRequest):
+    # Add your predict endpoint input schema here
+    # Follow the convention of <field_name>: <type_hint>
+    # Check https://pydantic-docs.helpmanual.io/usage/models/ for more info
+
+    # Optionally add validators for your features
+    # Follow the pydantic convention
+    # Check https://pydantic-docs.helpmanual.io/usage/validators/ for more info
+
+    pass
+
+
+# TODO: [2] Define your response schema
+class MyPredictionResponse(KonanServiceBasePredictionResponse):
+    # Add your predict endpoint output schema here
+    # Follow the convention of <field_name>: <type_hint>
+    # Check https://pydantic-docs.helpmanual.io/usage/models/ for more info
+
+    pass
+
+
+class MyModel(KonanServiceBaseModel):
+    def __init__(self):
+        super().__init__()
+        # TODO: [3] Initialize your model
+        # Add logic to initialize your actual model here
+        # Maybe load weights, connect to a database, etc ..
+
+        # For example, the following code will load a model saved as a model.pickle file in the models/ directory
+        # import os
+        # import pickle
+        # self.loaded_model = pickle.load(open(f"{os.environ['MODELS_DIR']}/model.pickle", 'rb'))
+
+    def predict(self, req: MyPredictionRequest) -> MyPredictionResponse:
+        # TODO: [4] Implement your prediction logic
+        # Optionally preprocess the request here
+        prediction = "" # Use your logic to make a prediction
+        # Optionally postprocess the prediction here
+        return prediction
+
+
+konan_service = KonanService(MyPredictionRequest, MyPredictionResponse, MyModel)
```

Comment:
Please rename `konan_service` to be `app` or `application` to conform to wsgi conventions.

Reply:
Will do